benchmark/single-source/ObjectiveCBridging.swift

```diff
@@ -773,3 +785,73 @@ public func run_UnicodeStringFromCodable(_ N: Int) {
   }
   #endif
 }
+
+#if _runtime(_ObjC)
+var bridgedArray:NSArray! = nil
+var bridgedArrayMutableCopy:NSMutableArray! = nil
+var nsArray:NSArray! = nil
+var nsArrayMutableCopy:NSMutableArray! = nil
+#endif
+
+public func setup_bridgedArrays() {
+  #if _runtime(_ObjC)
+  var arr = Array(repeating: NSObject(), count: 100) as [AnyObject]
+  bridgedArray = arr as NSArray
+  bridgedArrayMutableCopy = (bridgedArray.mutableCopy() as! NSMutableArray)
+  nsArray = NSArray(objects: &arr, count: 100)
+  nsArrayMutableCopy = (nsArray.mutableCopy() as! NSMutableArray)
+  #endif
+}
+
+@inline(never)
+public func run_BridgedNSArrayObjectAtIndex(_ N: Int) {
+  #if _runtime(_ObjC)
+  for _ in 0 ..< N {
+    for _ in 0..<50 {
```

Comment:
Is there a reason for the nested loops here? We usually just go from `0` to `50 * N` elsewhere. Overflows are only a concern on 32-bit platforms, and the benchmark runner can work around them by limiting the iteration count.

Reply:
oh, right. Just wasn't thinking :)